### DISCUSSION TITLE
Drag & Drop to the Taskbar [Feature Req]

### DISCUSSION THREAD

**Discussion by CrackHub** [ramensoftware/windhawk-mods#1351] [Mod ideas]
Windows 11 dropped support essential feature. Can you make a mod for this?
I know **Windows11DragAndDropToTaskbarFix** already fix this but it is laggy sometimes.

**Comment by m417z** [comment]
**Windows11DragAndDropToTaskbarFix** is only relevant for Windows 11 versions older than 22H2. From [the readme](https://github.com/HerMajestyDrMona/Windows11DragAndDropToTaskbarFix/blob/main/README.md):

> UPDATE: Windows 11 22H2 (build 22621 and 22622, released in September 2022) restores the drag & drop functionality natively, so this program is not longer needed. To learn how to update your Windows 11, please [visit this page](https://www.microsoft.com/software-download/windows11). You can still use this program if you're running an older version of Windows 11. In case you have troubles removing it, please read https://github.com/HerMajestyDrMona/Windows11DragAndDropToTaskbarFix/issues/81.

**Reply by CrackHub** [reply]
I'm using 26100.2605 fresh install.
Why I can't drag anything 🤔 Does disabling UAC cause problems?
![1](https://github.com/user-attachments/assets/0d6b457a-6c4f-4f27-af4a-8b47f87f4cba)

**Reply by m417z** [reply]
I don't know, but it works for me.

**Reply by CrackHub** [reply]
What is your "EnableLUA" key in this registry. I'm using 0
**HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\CurrentVersion\Policies\System**

**Reply by m417z** [reply]
Mine is 1.

**Reply by CrackHub** [reply]
Yeah I think it is cause problem. I don't want use UAC with 1, thats why I asked new feature if it can be done.

**Reply by m417z** [reply]
I can't think for a reason for it to be relevant to drag'n'drop. If you change it to 1, does it work for you?

**Reply by CrackHub** [reply]
Yes I try this now and worked. **making it 1 working**, but making it 0 disabling drag&drop. It is really annoying.
Also you can test with 0, need to restart windows.

**Reply by m417z** [reply]
Are you running explorer.exe as administrator?

**Reply by CrackHub** [reply]
I don't do anything, it is running when windows open.

Also Firefox issue with UAC disabled: https://support.mozilla.org/en-US/kb/windows-administrator-launcher-process-error-fix

**Reply by m417z** [reply]
OK, I googled a bit, and it seems to be a known limitation. [Terminal has an issue with it as well](https://github.com/microsoft/terminal/issues/6661), and it seems like there's no easy solution.
